elasticsearch/values.yaml

```diff
@@ -35,6 +35,8 @@ esConfig: {}
 #  log4j2.properties: |
 #    key = value
 
+createCert: true
```

Comment:
``true`` as default value is a big breaking change!

Reply:
Yes, with Elasticsearch 8.0.0 security will be enabled by default (https://github.com/elastic/elasticsearch/pull/77231). This PR update the chart to make it work with security by default. Note that this change is for the future 8.0.0 of the chart and will not be backported to the previous versions.